justfile

```
@@ -172,20 +172,20 @@ _build-bolt-boost:
 	cd bolt-boost && docker build -t ghcr.io/chainbound/bolt-boost:0.1.0 . --load
 
 # deploy the bolt sidecar to the dev server
-deploy-sidecar-dev:
-    chmod +x ./scripts/deploy_bolt_sidecar.sh && ./scripts/deploy_bolt_sidecar.sh
+deploy-sidecar-dev chain:
+    chmod +x ./scripts/deploy_bolt_sidecar.sh && ./scripts/deploy_bolt_sidecar.sh {{chain}}
 
 # Check the status of the sidecar service on the dev server
-status-sidecar-dev:
-    ssh shared@remotebeast "sudo systemctl status bolt_sidecar" | less
+status-sidecar-dev chain:
+    ssh shared@remotebeast "sudo systemctl status bolt_sidecar{{chain}}" | less
```

Probably a typo here

```suggestion
    ssh shared@remotebeast "sudo systemctl status bolt_sidecar_{{chain}}" | less
```

---

ty!